evap/static/ts/src/notebook.ts

```diff
@@ -1,17 +1,12 @@
 import "./translation.js";
-import { unwrap, assert, selectOrError } from "./utils.js";
-
-const NOTEBOOK_LOCALSTORAGE_KEY = "evap_notebook_open";
-const COLLAPSE_TOGGLE_BUTTON_SELECTOR = "#notebookButton";
-const WEBSITE_CONTENT_SELECTOR = "#evapContent";
-const NOTEBOOK_FORM_SELECTOR = "#notebook-form";
+import { unwrap, assert } from "./utils.js";
 
 class NotebookFormLogic {
     private readonly notebook: HTMLFormElement;
     private readonly updateCooldown = 2000;
 
-    constructor(notebookFormId: string) {
-        this.notebook = selectOrError(notebookFormId);
+    constructor(notebook: HTMLFormElement) {
+        this.notebook = notebook;
     }
```

Comment:
You can now remove the variable declaration and move it here:
```suggestion
    constructor(private readonly notebook: HTMLFormElement) {}
```